config/environments/development.rb

```diff
@@ -43,6 +43,7 @@
   end
 
   # Generate random VAPID keys
+  # Uses Webpush class, Webpush::Legacy does the same thing.
```

Comment:
I think this comment can be removed, `Webpush::Legacy` (as in, in the gem maintained by Mastodon) does not cover this at all.